www-apps/zeppelin-bin/zeppelin-bin-0.11.2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 verify-sig
+
+MY_PN="zeppelin"
+MY_P="${MY_PN}-${PV}-bin-all"
+
+DESCRIPTION="Web-based interactive data analytics notebook launcher"
+HOMEPAGE="https://zeppelin.apache.org"
+SRC_URI="mirror://apache/zeppelin/${MY_PN}-${PV}/${MY_P}.tgz -> ${P}.tgz
+	verify-sig? ( https://downloads.apache.org/zeppelin/zeppelin-${PV}/${MY_P}.tgz.asc -> ${P}.tgz.asc )"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0"
```

Comment:
The license is incomplete.  The `LICENSE` file contains the text of Apache 2, and then a few other licenses.  Most probably they should be added here.  Also there is a `licenses` directory installed, it should be removed if `LICENSE` is fixed .

Reply:
> The license is incomplete. The `LICENSE` file contains the text of Apache 2, and then a few other licenses. Most probably they should be added here. Also there is a `licenses` directory installed, it should be removed if `LICENSE` is fixed .

Must have been missing already in the older version.  Is there a certain syntax for licenses of such bundled dependencies? Couldn't find anything documented.

Reply:
Nothing special.  Just add them all, something like `LICENSE="Apache-2.0 BSD CDDL etc"`

Looking at rust ebuilds, it appeares the main license is listed first, then the rest of them are sorted alphabetically.  Not sure if there is a rule about sorting them.

Reply:
> Looking at rust ebuilds, it appeares the main license is listed first, then the rest of them are sorted alphabetically. Not sure if there is a rule about sorting them.

I've added what I could find. But comparing to typical bundleware packages like `app-text/languagetool` or `dev-util/android-sdk-update-manager` it feels wrong.